business/tls.go

```diff
@@ -0,0 +1,188 @@
+package business
+
+import (
+	"fmt"
+	"sync"
+
+	"github.com/kiali/kiali/kubernetes"
+	"github.com/kiali/kiali/models"
+)
+
+type TLSService struct {
+	k8s           kubernetes.IstioClientInterface
+	businessLayer *Layer
+}
+
+const (
+	MTLSEnabled          = "MTLS_ENABLED"
```

Comment:
Ok, I see that there is a refactoring of these from istio_config to this tls business.
But it seems the constants has changed, I think this needs to be propagated to the UI side as well.